Packages/vcs/vcs/utils.py

```diff
@@ -1666,13 +1666,25 @@ def creategraphicsmethod(gtype, gname='default', name=None):
     return copy_mthd
 
 
+# Returns the float value for datawc_...
+# datawc_ can be a float or a cdtime.reltime
+# TODO: Investigate why datawc is converted to a cdtime.reltime
+def getDataWcValue(v):
+    if (type(v) is type(cdtime.reltime(0, 'months since 1900'))):
```

Comment:
ok. A `comptime` is a componentTime. Basically a date. Whereas `reltime` is a relative time, i.e a delta since some reference date. You should take a look at: http://uvcdat.llnl.gov/documentation/cdms/cdms_3.html